internal/runners/exec/exec.go

```diff
@@ -188,8 +188,8 @@ func (s *Exec) Run(params *Params, args ...string) (rerr error) {
 	lang := language.Bash
 	scriptArgs := fmt.Sprintf(`%q "$@"`, exeTarget)
 	if strings.Contains(s.subshell.Binary(), "cmd") {
-		lang = language.PowerShell
-		scriptArgs = fmt.Sprintf("& %q @args\nexit $LASTEXITCODE", exeTarget)
```

Comment:
I reverted this because it's no longer needed, and for whatever reason this combined with the new solution I pushed caused a regression. I suspect because we were chaining cmd > ps1 > bat, whereas now it's just cmd > bat > bat.